support/jwt/src/main/kotlin/com/xorker/xpr/support/jwt/SignatureKey.kt

```diff
@@ -0,0 +1,7 @@
+package com.xorker.xpr.support.jwt
+
+import java.security.Key
+
+interface SignatureKey {
+    fun getKey(): Key
```

Comment:
위에서 말한 방법 적용하면 아래처럼!
```suggestion
    val key: Key
```